### ISSUE TITLE
Use pycharm to import clingo on windows

### ISSUE THREAD

**Issue by liuhy397** [potassco/clingo#113]
thank you for help me with my problem,I want to import clingo in my pycharm on windows,but the only version of clingo is 5.21,And i can't import it,so could you give me some advice,thank you

**Comment by rkaminsk** [comment]
The easiest way to install clingo on windows is using anaconda. A quick search revealed that conda is supported by pycharm 5.

<https://blog.jetbrains.com/pycharm/2015/10/pycharm-5-eap-143-24-is-available/https://blog.jetbrains.com/pycharm/2015/10/pycharm-5-eap-143-24-is-available/>
<https://www.jetbrains.com/help/pycharm/5.0/conda-support-creating-conda-environment.html>
<https://anaconda.org/potassco/clingo>

**Comment by liuhy397** [comment]
![image](https://user-images.githubusercontent.com/39975865/41325025-9c4f8b40-6eea-11e8-9668-3c64fd7e3ad7.png)
thank you for help me with my problem,thank you very much,I have install anaconda,and install clingo5.22 in my anaconda,In my pycharm,I already have clingo in my project interpretor,but i sitll can not import clingo,it  tell me dll load failed,can not find specified module.
Thank you again for help me.

**Comment by liuhy397** [comment]
thank you

**Comment by liuhy397** [comment]
![image](https://user-images.githubusercontent.com/39975865/41325665-1cd9c494-6eee-11e8-833e-21f20367db72.png)
I paste two file in my DLLS directionary and I do not know if they are correct,one named clingo.dll,and anather named clingo.cp36-win_amd64.pyd,  I sdudy the example of python api,but it do not have a result,thank you for help me.
![image](https://user-images.githubusercontent.com/39975865/41325918-5e0d2f72-6eef-11e8-9264-4c57a005ea05.png)
I am a newer for programming,so thank you again for help me

**Comment by rkaminsk** [comment]
I do not know why you had to copy the dlls yourself. When using python from anaconda, this should not be necessary. Maybe your conda environment was not properly activated. I can look into this next week.

In your code, you do not call the main function so nothing is executed. Also if you are using the python module, you have to take care of the output yourself. Take a look at the following example program:

```python
from textwrap import dedent

import clingo

# create a control object
ctl = clingo.Control()

# add a logic program
ctl.add("base", [], dedent("""\
    {a}.
    b :- a.
    c :- not a.
    """))
# ground the added program
ctl.ground([("base", [])])

# configure solver to print all models
ctl.configuration.solve.models = 0
# start solving
with ctl.solve(yield_=True) as handle:
    # loop over all models and print them
    for model in handle:
        print (model)
```

**Comment by liuhy397** [comment]
firstly,thank you for your help,this example run successful and I feel so happy,without your help,I will never know how to do it,thank you again.

**Comment by rkaminsk** [comment]
Hi, I tested with pycharm and anaconda on windows. It turns out that pycharm only adds some of the paths that conda's activate would add. I restructured the clingo windows package slightly to accommodate for this. With the latest clingo version (only the development version for now) everything should work out of the box without copying any dlls. With the next clingo 5.3 release, things will also work for packages with the main label.
